src/shared/options.ts

```diff
@@ -511,4 +515,5 @@ export const LanguageServerOptionsThatTriggerReload: ReadonlyArray<keyof Languag
     'componentPaths',
     'enableXamlTools',
     'useServerGC',
+    'reportInformationAsHint',
```

Comment:
Changing 'reportInformationAsHint' will prompt for the user to reload the workspace